README.md

```diff
@@ -14,6 +14,27 @@ The steps below detail the installation of this site locally for development.
 yarn
 ```
 
+### Pre-commit Setup
+
+This project uses pre-commit hooks to ensure code quality. To set up pre-commit:
+
+1. Install pre-commit:
```

Comment:
I think the friction here is it doesn't happen automatically on npm install. I definitely have no problem installing this, just wasn't aware of it.

That said since this is a JS project, I've typically used [Husky](https://typicode.github.io/husky/), which installs via npm, so one less thing for an internal or external contributor to think about and it will run without any additional setup.

Reply:
Yeah, I agree, and husky + a cspell npm dependency was my original intent until I realized what was already setup. If we add husky based hooks and the cspell npm dependency, i'd prefer to remove the existing pre-commit stuff, so the cspell versions are consistent. Then, would netlify would run the linting though. I do like the separate lint checks, so we get a preview regardless of linting failures.